packages/nextjs/src/tools/codegen/import-map.ts

```diff
@@ -0,0 +1,413 @@
+import * as ts from 'typescript';
+import path from 'path';
+import fs from 'fs';
+import { debug } from '@sitecore-content-sdk/core';
+import { getComponentList } from '@sitecore-content-sdk/core/tools';
+import { SitecoreConfig } from '@sitecore-content-sdk/core/config';
+import crypto from 'crypto';
+import { defaultImportEntries } from './default-import-map';
```

Comment:
This way of integrating default map creates dependency between a generated map and default one. So user can't access a default map unless they like our heuristics for generated map. 
Integration point could be moved to later point, essentially to DesignLibrary.tsx component.

Reply:
I was convinced we combine the two in package - but this makes sense. Adjusted the thing and we now combine them in generated file.

Reply:
but it seems in previous implementation you serialized it into generated import map, or did I understood it wrongly?